src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -469,3 +479,10 @@ def remove_docker_image(repo: str, tag: str, dc: DockerClient):
         fkwargs={"force": True},
         tries=5,
     )
+
+
+def remove_container_volumes(container: Container, dc: DockerClient):
+    """Remove volumes related to the given container"""
```

Comment:
Leave note here about the fact that container should be already removed when we call this method

Reply:
Done.